src/inference/mh.jl

```diff
@@ -25,6 +28,18 @@ function MH(space...)
             elseif s[2] isa Function
                 push!(props, AMH.StaticProposal(s[2]))
             end
+        elseif length(space) == 1
+            # If we hit this block, check to see if it's 
+            # a run-of-the-mill proposal or covariance
+            # matrix.
+            prop = if s isa AdvancedMH.Proposal
+                s
+            elseif typeof(s) <: Array{<:Real, 2}
+                AdvancedMH.RandomWalkProposal(MvNormal(s))
+            end
```

Comment:
Maybe it would be nicer and easier to extend if one defines
```julia
proposal(p) = p
proposal(cov::AbstractMatrix) = AdvancedMH.RandomWalkProposal(MvNormal(cov))
```
and then just uses
```suggestion
        elseif length(space) == 1
            # Extract or generate proposal.
            prop = proposal(s)
```

Reply:
Okay, I added this. Good comment.